packages/site/pages/index.tsx

```diff
@@ -1,44 +1,14 @@
-import Image from "next/image";
-import { useRouter } from "next/router";
-import Layout from "../components/Layout";
+import { BuilderComponent } from '@builder.io/react';
+import Layout from '../components/Layout';
 
 export const Index = () => {
-  const router = useRouter();
-
   return (
     <Layout hideHeaderAndFooter>
       <div
-        style={{ marginLeft: "calc(50% - 50vw)" }}
-        className="h-screen w-screen flex items-center justify-center pb-10 bg-dark"
+        style={{ marginLeft: 'calc(50% - 50vw)' }}
+        className="flex items-center justify-center w-screen h-screen pb-10 bg-dark"
       >
-        <div className="m-auto w-11/12 max-w-4xl flex-col flex">
-          <Image
-            onClick={() => {
-              if (window.location.hostname === "localhost") {
-                router.push("/discover");
-              }
-            }}
-            src="https://cdn.builder.io/api/v1/image/assets%2FYJIGb4i01jvw0SRdL5Bt%2Fd0a71921f7c74e35b9b9c6938de22bae"
-            width="2490"
-            height="1182"
-            alt="logo"
-          />
-
-          <a
-            rel="noreferrer"
-            href="mailto:has-invite@builder.io?subject=Please%20add%20me%20to%20the%20waitlist%20for%20HeadlessApp.Store&body=Thanks!"
-            className="text-center text-offwhite block mx-auto px-6 py-4 border-offwhite border-2 rounded"
-          >
-            Request early access
-          </a>
-          <a
-            rel="noreferrer"
-            href="mailto:has-partners@builder.io"
-            className="text-center text-offwhite block mx-auto px-6 py-4 border-offwhite opacity-50 rounded mt-4"
-          >
-            Request to be a partner
-          </a>
-        </div>
+        <BuilderComponent model="page" />
```

Comment:
we'll want this to render server side so we'll need to add `getStaticProps` and pass the content to this as `content={content}`

Reply:
[this](https://github.com/BuilderIO/builder/blob/master/examples/next-js-simple/pages/%5B%5B...path%5D%5D.tsx#L84) shows an example of that, passing renderLink helps ensure client side routing too

Reply:
Maybe the SSR change, I'm not sure I understand the renderLink suggestion as this project doesn't have a Link component defined, would it just be passing the Link component provided by @builder.io/react ?